pypeit/traceslits.py

```diff
@@ -1254,7 +1254,7 @@ def load_master(self, filename, force = False):
 
         """
         # Does the master file exist?
-        if not (os.path.isfile(filename + '.fits') & os.path.isfile(filename + '.json')):
+        if not (os.path.isfile(filename + '.fits.gz') & os.path.isfile(filename + '.json')):
```

Comment:
I thought you turned off gzip for these files.

Reply:
I turned it back on.